docs/vclusters/clariden.md

```diff
@@ -9,63 +9,73 @@
 ## Cluster Specification
 ### Hardware
 Clariden consists of ~1200 [Grace-Hopper nodes][ref-alps-gh200-node]. Most nodes are in the [`normal` slurm partition][ref-slurm-partition-normal], while a few nodes are in the [`debug` partition][ref-slurm-partition-debug].
+The nodes are interconnected with the [slingshot high speed network][ref-alps-slingshot-network].
 
+As usual the login nodes have direct internet connections, while the compute nodes use a [proxy server][ref-alps-slingshot-network] to access the internet.
+!!! todo
+    Document proxy and the implications (normally transparent, but git needs modifications
+    Fix the ref
 
+### File systems and storage
+The scratch filesystem is hosted on [IOPStore][ref-storage-iopstor], but also the capacity storage [Capstor][ref-storage-capstor] is mounted at `/capstor/scratch/cscs`.
+The variables `STORE` and `PROJECT` are not set on Clariden.
+The home directory is hosted on [VAST][ref-storage-vast].
 
-!!! todo
-    a standardised table with information about
+As usual, an overview of your quota on the different filesystems, can be obtained by the `quota` command.
 
-    * number and type of nodes
+## Getting started
+The project and resources are managed by [this tool][ref-account-waldur].
 
-    and any special notes
+### Connect to Clariden
+You can connect to Clariden via [ssh][ref-ssh-config], ensuring that the file `~/.ssh/config` has these settings (replace `cscsusername` with your username).
 
-## Logging into Clariden
+```title="$HOME/.ssh/config"
+Host ela
+    HostName ela.cscs.ch
+    User cscsusername
+    IdentityFile ~/.ssh/cscs-key
 
-!!! todo
-    how to log in, i.e. `ssh clariden.cscs.ch` via `ela.cscs.ch`
+Host clariden
+    HostName clariden.alps.cscs.ch
+    ProxyJump ela
+    User cscsusername
+    IdentityFile ~/.ssh/cscs-key
+    IdentitiesOnly yes
+```
```

Comment:
Link to https://eth-cscs.github.io/cscs-docs/access/ssh/ or more specifically https://eth-cscs.github.io/cscs-docs/access/ssh/#logging-in?